content/apps/managed-services.md

```diff
@@ -25,54 +21,42 @@ The v1 services in [cf-services-contrib-release](https://github.com/cloudfoundry
 
 	brew install jq
```

Comment:
No idea what `jq` might have been used for... This must be some ancestral remnant. For CLI installation we should point users to the [setup page](https://docs.cloud.gov/getting-started/setup/).